src/app/carbon-estimator-form/carbon-estimator-form.component.html

```diff
@@ -235,9 +235,10 @@
 
   <div class="tce-flex tce-gap-4 tce-flex-row-reverse">
     <button
-      class="tce-button-calculate tce-px-3 tce-py-2 disabled:tce-opacity-50 disabled:tce-cursor-not-allowed"
+      class="tce-button-calculate tce-px-3 tce-py-2"
+      [ngClass]="!estimatorForm.valid ? 'tce-opacity-50 tce-cursor-not-allowed' : ''"
```

Comment:
```suggestion
      [ngClass]="{ 'tce-opacity-50 tce-cursor-not-allowed': !estimatorForm.valid }"
```
Just a minor thing, this way avoids the ternary/empty string.

Reply:
Ah ok, thanks! I've made that change.